libmscore/tuplet.cpp

```diff
@@ -694,7 +694,8 @@ void Tuplet::read(XmlReader& e)
             else if (!DurationElement::readProperties(e))
                   e.unknown();
             }
-      Fraction f(_ratio.reduced().denominator(), _baseLen.fraction().denominator());
+      Fraction r = (_ratio == 1) ? _ratio : _ratio.reduced();
+      Fraction f(r.denominator(), _baseLen.fraction().denominator());
```

Comment:
Ah yes, you're right.
Maybe we could add in the compatibility code a sanity check for the tuplets which computes the duration and evaluates the baselen, similar to what is done in MusicXML import here: https://github.com/musescore/MuseScore/blob/d85ff2851b8f6a075c6d7b31549611942e86ab05/mscore/importmxmlpass2.cpp#L969
and possibly corrects the values during the import.